scripts/generate_linux_tests.rb

```diff
@@ -96,10 +96,10 @@ def createLinuxMain(testsDirectory, allTestSubDirectories, files)
 
```

Comment:
Unrelated to the PR but is this script still needed?

Reply:
recent versions of swift-pm have experimental feature `--enable-test-discovery` that make this obsolete, will fix in separate PR